client/ayon_maya/plugins/publish/extract_layout.py

```diff
@@ -106,7 +106,7 @@ def process(self, instance):
             json_path = os.path.join(stagingdir, json_filename)
             t_matrix = []
             with lib.maintained_selection():
-                cmds.ls(asset, dagObjects=True)
+                cmds.select(asset, noExpand=True)
                 sel = om.MGlobal.getActiveSelectionList()
                 dagpath = sel.getDependNode(0)
```

Comment:
This may be even better as:
```suggestion
                sel = om.MSelectionList()
                sel.add(asset)
                dagpath = sel.getDependNode(0)
```

That way you don't even need to make any selection in the scenes and you can also remove the maintained selection.